File: src/main/java/ejerciciosJava/Ejercicio4.java
package ejerciciosJava;

import java.util.Random;

public class Ejercicio4 {
    public static void main(String[] args) {
        Random random = new Random();
        int generatedNum = random.nextInt(100) + 1;
        int userNum;

        do {
            userNum = Utils.pedirEntero("Adivina un número del 1 al 100: ");
            if (userNum > generatedNum) {
                System.out.println("Casii, el número a adivinar es más bajo");
            } else if (userNum < generatedNum) {
                System.out.println("Casii, el número a adivinar es más alto");
            }
        } while (generatedNum != userNum);

        System.out.println("Acertaste");
    }
}

File: src/main/java/ejerciciosKotlin/Ejercicio4.kt
package ejerciciosKotlin

import kotlin.random.Random

fun ejercicio4(){
    val generatedNum = Random.nextInt(1,100)
    var userNum:Int? = null
    do {
        userNum = pedirEntero("Adivina un número del 1 al 100: ")
        if (userNum > generatedNum){
            println("Casii, el número a adivinar es más bajo")
        } else if (userNum < generatedNum){
            println("Casii, el número a adivinar es más alto")
        }
    }while (generatedNum != userNum)
    println("Acertaste")
}